myuw/templates/handlebars/card/international_student.html

```diff
@@ -4,8 +4,71 @@
     <h3 class="myuw-card-title">
         International Student
     </h3>
+    <div class="links-container">
+        <h4 class="myuw-card-heading">Visa and Immigration Rules</h4>
 
-    <div class="myuw-card-section last">
+        <ul class="unstyled-list">
+        {{#if seattle}}
+	    <li><a href="https://iss.washington.edu/travel-visas/" target="_blank" data-linklabel="Travel and visas for Int'l students">Travel and Visas</a></li>
```

Comment:
Hi Hyunju, why these links are using https?  Could you check if just http works?

Reply:
I copy & paste the URL from ISS and CIE site. 
http reroutes to https.
Many of UW dept sites are 'https://....' nowadays and that trend will continue.